docs/versioned_docs/version-2.16/workflows/terraform-provider.md

```diff
@@ -78,6 +78,17 @@ This example shows how to set up a Constellation cluster with the reference IAM
 
   Optionally, you can prefix the `terraform apply` command with `TF_LOG=INFO` to collect [Terraform logs](https://developer.hashicorp.com/terraform/internals/debugging) while applying the configuration. This may provide helpful output in debugging scenarios.
   </tabItem>
+  <tabItem value="stackit" label="STACKIT">
```

Comment:
It will as a special case with the 2.16.1 release. Since STACKIT support is just launching, there is no real issue with adding those features in a patch release.
Also, adding a new minor release forces users on other CSPs to upgrade without getting any new features.